adding examples for the new windows_user_privilege and windows_securi… (#2488)
chef/chef-web-docs: content/resources/windows_user_privilege/_index.md

  :remove:
    markdown: Remove a user privilege
  :set:

>     markdown: This will remove any user not listed in the `users` field and only allow what is listed what was provided to remain.

Reviewer: ```suggestion
    markdown: Set the privileges that are listed in the `privilege` property for only the users listed in the `users` property.
```

  description_list:
  - markdown: An optional property to set the privilege for given users. Use only
      with set action.
examples: |

>   **Enforce only Builtin Administrators Group and Authenticated Users allowed for the SeNetworkLogonRight Privilege**:

Reviewer: ```suggestion
  **Set the SeNetworkLogonRight Privilege for the Builtin Administrators Group and Authenticated Users**:
```

  ```ruby
  windows_user_privilege 'Netowrk Logon Rights' do
    privilege      'SeNetworkLogonRight'
    users          ['BUILTIN\Administrators', 'NT AUTHORITY\Authenticated Users']
    action         :set
  end
  ```

>   **Add Builtin Guests and Local Accounts to the SeDenyRemoteInteractiveLogonRight Privilege**:

Reviewer: ```suggestion
  **Add the SeDenyRemoteInteractiveLogonRight Privilege to the Builtin Guests and Local Accounts User Groups**:
```

  ```ruby
  windows_user_privilege 'Remote interactive logon' do
    privilege      'SeDenyRemoteInteractiveLogonRight'
    users          ['Builtin\Guests', 'NT AUTHORITY\Local Account']
    action         :add
  end
  ```

>   **Enforce only Builtin Guests and Adminiistrator Gruops allowed for SeCreatePageFile Privilege**:

Reviewer: ```suggestion
  **Provide only the Builtin Guests and Administrator Groups with the SeCreatePageFile Privilege**:
```

  ```ruby
  windows_user_privilege 'Create Pagefile' do
    privilege      'SeCreatePagefilePrivilege'
    users          ['BUILTIN\Guests', 'BUILTIN\Administrators']
    action         :set
  end
  ```

>   **Remove Builtin Guests from SeCreatePageFile Privilege**:

Reviewer: ```suggestion
  **Remove the SeCreatePageFile Privilege from the Builtin Guests Group**:
```
